Detectors/Calibration/include/DetectorsCalibration/TimeSlotCalibration.h

```diff
@@ -75,14 +76,24 @@ class TimeSlotCalibration
 
   virtual void print() const;
 
+  void setCurrentTFInfo(uint32_t firstOrbit, uint32_t tfCounter, uint32_t runNumber, uint64_t creation);
+  auto getTFfirstOrbit() const { return mTFfirstOrbit; }
+  auto getTFcounter() const { return mTFcounter; }
+  auto getTFrunNumber() const { return mTFrunNumber; }
+  auto getTFcreationTime() const { return mTFcreationTime; }
+
  protected:
   auto& getSlots() { return mSlots; }
 
- private:
   TFType tf2SlotMin(TFType tf) const;
 
   std::deque<Slot> mSlots;
 
+  uint32_t mTFfirstOrbit = 0;
+  uint32_t mTFcounter = 0;
+  uint32_t mTFrunNumber = 0;
+  uint64_t mTFcreationTime = 0;
```

Comment:
@martenole you can identify each TF slot in the residuals tree by mTFfirstOrbit and mTFcreationTime